fanuc:
  table_detection_area:
    ros__parameters:
      robot_name: fanuc
      location: table
      video_stream: http://192.168.1.104/mjpg/video.mjpg
      publish_frames: True
      background_value_threshold: 190
      transform:
        parent_frame: world
        child_frame: fanuc_table_image
        rotation:
          roll: 0.0
          pitch: 3.14159
          yaw: 0.0
        translation:
          x: 1.0414 # 41"
          y: 0.381 # 15"
          z: 0.0

  conveyor_detection_area:
    ros__parameters:
      robot_name: fanuc
      location: conveyor
      base_frame: fanuc_base_link
      video_stream: http://192.168.1.108/mjpg/video.mjpg
      publish_frames: True

motoman:
  table_detection_area:
    ros__parameters:
      robot_name: motoman
      location: table
      video_stream: http://192.168.1.110/mjpg/video.mjpg
      publish_frames: True
      background_value_threshold: 138
      transform:
        parent_frame: world
        child_frame: motoman_table_image
        rotation:
          roll: 0.0
          pitch: 3.14159
          yaw: 0.0
        translation:
          x: 1.6764 # 66"
          y: 0.127 # 5" 
          z: 0.0

  conveyor_detection_area:
    ros__parameters:
      robot_name: motoman
      location: conveyor
      base_frame: motoman_base_link
      video_stream: http://192.168.1.107/mjpg/video.mjpg
      publish_frames: True
      rotation_from_base_frame: 0.0
      table_origin:
        x: 0.278
        y: 0.203
        z: -0.0254

teach:
  teach_detection_area:
    ros__parameters: 
      robot_name: teach
      location: table
      video_stream: http://192.168.1.104/mjpg/video.mjpg
      publish_frames: False